src/docs/product/performance/index.mdx

```diff
@@ -45,3 +45,17 @@ Below the graphs are a list of transactions that may be filtered down based on g
 ## Key Transaction View
 
 If you have transactions you frequently return to; you can mark each one as a key transaction in their corresponding [summary views](/product/performance/transaction-summary/#key-transaction). This enables a shortcut to view only Key Transactions by toggling the tab in the upper right corner of the homepage.
+
+## Trends View
+
+Trends surfaces transactions that have had significant changes in their performance over time. This view can be found by toggling the tab in the upper right corner of the homepage.
+
+![trends Homepage](trends-homepage.png)
+
+Trending transactions are calculated first by filtering out transactions with large fluctuations in [throughput](/product/performance/metrics/#throughput-total-tpm-tps), then determining the baselines of the two halves of the duration, and calculating their percentage change. Regressions are any percentage changes greater than 0, while improvements are percentage changes less than 0.
+
+For example looking at a transaction over two weeks with the default [P50](/product/performance/metrics/#average-transaction-duration) baseline, if the first week had a `p(50)=100s`, and the second a `p(50)=110s` the percentage is `(110/100)-1 = 0.1`. This means that this example transaction had a performance regression of `10%`.
+
+The accuracy of a trend improves the more data that is available, for this reason, trends perform best on long periods, and with transactions with large counts.
+
+While the baseline defaults to P50 but the Display dropdown lets you change this. The baseline is used to determine the percentage change but is also used for the comparison view found in the dropdown for each transaction. This comparison view finds the transaction in each period with the duration closest to the baseline and compares their spans.
```

Comment:
The first sentence is a little confusing.

`While we default the baseline calculation to use P50 you can change it via the dropdown, other options include P75, P95, P99 and average.`